### PR TITLE
TE-4572 Add PHP74 and new split checks

### PR DESCRIPTION
**Pull request by dereuromark**
- Developer(s): @dereuromark

- Ticket: https://spryker.atlassian.net/browse/TE-4572

- Release: https://release.spryker.com/release/pull-request-sdk/Spryk/35

#### Please confirm

- [ ] No new OS components - or they have been approved by legal department

#### Documentation

- [ ] Functional documentation provided or in progress?
- [ ] Integration guide for projects provided or not needed?
- [ ] Migration guides for all contained majors provided or not needed?

#### Release Table

   Module                | Release Type         | Constraint Updates         |
   :--------------------- | :------------------------ | :--------------------- |
   Spryk                 | patch (currently 0.x)  |                       |

#### Release Notes

{skip}

-----------------------------------------

##### Change log

Bugfixes

Fixed signatures of Commands.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -35,6 +35,7 @@ before_install:
 before_script:
   - if [[ $PREFER_LOWEST != 1 ]]; then composer install --no-interaction; fi;
   - if [[ $PREFER_LOWEST == 1 ]]; then composer update --prefer-lowest --no-interaction --prefer-dist --prefer-stable; fi;
+  - if [[ $PREFER_LOWEST == 1 ]]; then composer require --dev dereuromark/composer-prefer-lowest; fi
```
Comment (Incognito) [inline .travis.yml]: Is there a way to pin the version?

Comment (dereuromark) [reply .travis.yml]: No need to.

Comment (Incognito) [reply .travis.yml]: The dependencies of a build should be idempotent

Comment (dereuromark) [reply .travis.yml]: This is my tool, I control it.

**Pull request merged**